scripts/onboarding/managed/enable-monitoring.ps1

```diff
@@ -62,11 +62,10 @@ $isArcK8sCluster = $false
 $isAksCluster = $false
 $isUsingServicePrincipal = $false
 
-# released chart version in mcr
-$mcr = "mcr.microsoft.com"
-$mcrChartVersion = "2.8.3"
-$mcrChartRepoPath = "azuremonitor/containerinsights/preview/azuremonitor-containers"
-$helmLocalRepoName = "."
+# microsoft helm chart repo
+$microsoftHelmRepo="https://microsoft.github.io/charts/repo"
```

Comment:
Are we already pushing the helm chart here in our release process? Will we not push the helm chart to the mcr.microsoft.com anymore then?

Reply:
Yes, the same repo - https://microsoft.github.io/charts/repo used for on-prem k8s which we already have the release process. I have added note in the release process doc to indicate the same repo being used for AROv4 as well.  Yes, we dont use mcr.microsoft.com or our ACR as chart repo either for On-prem K8s or AROv4.